packages/algoliasearch-helper/src/requestBuilder.js

```diff
@@ -4,9 +4,7 @@ var merge = require('./functions/merge');
 
 function sortObject(obj) {
   return Object.keys(obj)
-    .sort(function (a, b) {
-      return a.localeCompare(b);
-    })
```

Comment:
I'm guessing you removed it because it didn't bring any value as we just need consistency in the sorting and not ensuring numbers are correctly sorted, or was there other concerns?

Reply:
localeCompare isn't about numbers, but about things like æ being sorted at the end of the alphabet in some languages. We don't really need that, and everything behaves better if we use the default sort (it's also much faster)

Reply:
thanks for the clarification!